README.md

```diff
@@ -1,5 +1,345 @@
 # Quansight Website
 
+## How to make changes to the website
+
+Before reading this section, familiarize yourself with [Vercel
+environments](https://vercel.com/docs/concepts/deployments/environments).
+
+This repo contains two websites (Consulting and Labs), but the process to update
+either is essentially the same.
+
+There are primarily two types of website changes, each with its own process:
+
+- Content changes (Storyblok)
+- Code changes (this GitHub repo)
+
+Note that Labs blog posts are a bit of an exception. Categorically they are
+content changes, but the content lives in the Git repo, so technically they are
+code changes, so they follow the process for code changes.
+
+This section will cover the process for each type of change.
+
+### Content changes (Storyblok)
+
+Content in Storyblok moves through several stages: drafting, reviewing, ready to
+publish, published, deployed. It's confusing, but "published" in our Storyblok
+configuration does not actually mean that the content has gone live on the
+corresponding public website. However, you should never publish any content in
+Storyblok if it is not ready for public presentation. In our configuration,
+publishing content in Storyblok is a signal to the rest of the team that says:
+this content is ready at any time to appear on the public website. If it's not
+ready, don't hit the publish button.
+
+The Storyblok process is covered in more detail in the GitHub repo's wiki. A
+simplified version is presented here.
+
+The process for creating, updating, or deleting content in Storyblok is the
+same. There are three major stages, each with its own steps.
+
+1. Editing
+   1. Make your changes in Storyblok.
+   2. Preview how your changes will look on the site using Storyblok's preview
+      iframe. You should see a yellow banner at the top of the page telling you
+      that you can see unpublished content.
+   3. Save your changes.
+2. Publishing
+   1. Subscribe to Slack channel `#vercel-notifications` if you are not already
+      subscribed.
+   2. Once your content changes are approved, click the publish button in
+      Storyblok. Remember, clicking "publish" means that you are certifying that
+      to the best of your knowledge, these changes should be ready to be
+      deployed at any time to the live website. When you click the publish
+      button, Storyblok will make a request to Vercel to start a preview
+      deployment.
+   3. Look for deployment notifications in Slack. Vercel will send a message to
+      the `#vercel-notifications` channel containing a link to the preview URL.
+   4. Visit the Vercel preview URL to review and double-check your changes.
+3. Going live
+   1. When you want your changes to go to the live production site, coordinate
```

Comment:
Is this merge to `main` still going to be mediated by an entry in {CONTENT_CHANGELOG}?

Reply:
The librarian in me definitely wants this to be a part of the process. 

The developer/code-shipper in me worries about it slowing the team down. It adds a step to onboarding content people. It adds cognitive overhead: for example, if I make a tiny typo fix, say adding an "s" to the end of one word on a page, do I need to add that to the content log? 

I think this points to a flaw in the design decision I describe in this document for hotfixes. The suggestion is to add an entry to the change log. But maybe instead of calling it a change log, it should just be called a hotfix log?

@trallard do you have any thoughts on a content plus code change log? 

My biggest concern is the limited time that dev team members have. And this potentially adds a non-trivial step to the process: when merging develop into main, a developer has to make sure the change log reflects both the code changes in the merge pull request, as well as the content changes in Storyblok. But those content changes are not visible from within the GitHub UI. And there's no summary view of them in Storyblok either, so that requires either (a) a very diligent content team that always updates the change log with content changes, or (b) it requires the developer to reach out to someone on the content team to fill in the log. I dunno. I'm worried that the longer a deploy takes, the more cumbersome it is, the less inclined anyone will be towards wanting to do it. So maybe the rule would have to be something like: content editors are responsible for updating the content log on GitHub whenever they make changes. 

Actually... oh hm, idea: instead of hooking up the "publish" button in Storyblok to a Vercel webhook, require an update to the change log in order to kick off a deploy with the latest published content? Ah but if the content team decides to essentially queue up content by hitting publish over a span of days and only then decides to update the change log but in the meantime the code team wants to push some code to the main branch, now we're back in the same problem again.